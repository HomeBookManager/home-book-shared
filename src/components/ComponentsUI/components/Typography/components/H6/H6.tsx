import { FC } from 'react';

// components
import Typography from '../../Typography';

// types
import { TypographyFontType } from '../../enums';
import { TOmittedTypographyProps } from '../../types';

const H6: FC<TOmittedTypographyProps> = ({ children, ...restProps }) => (
  <Typography fontType={TypographyFontType.h6} {...restProps}>
    {children}
  </Typography>
);

export default H6;
