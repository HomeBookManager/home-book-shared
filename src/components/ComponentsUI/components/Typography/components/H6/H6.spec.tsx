import { render } from '@testing-library/react';

// components
import H6 from './H6';

describe('H6 snapshots', () => {
  it('should render H6', () => {
    //before
    const { asFragment } = render(<H6>H6</H6>);

    // result
    expect(asFragment()).toMatchSnapshot();
  });
});
