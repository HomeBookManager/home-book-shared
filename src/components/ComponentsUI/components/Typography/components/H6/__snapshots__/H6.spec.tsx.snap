// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H6 snapshots should render H6 1`] = `
<DocumentFragment>
  <h6
    class="Typography Typography--regular Typography--normal Typography--h6"
    data-test-text=""
  >
    H6
  </h6>
</DocumentFragment>
`;
